setup.py

```diff
@@ -112,9 +112,12 @@ def get_git_sha():
         'hive': [
             'pyhive[hive]>=0.6.1',
             'tableschema',
+            'thrift>=0.11.0, <1.0.0',
         ],
         'presto': ['pyhive[presto]>=0.4.0'],
         'gsheets': ['gsheetsdb>=0.1.9'],
+        'mysql': ['mysqlclient==1.3.13'],
```

Comment:
Any reason to pin these here? Note we may need to have a `requirements.in` file including the line`-e .` (and possibly mention of the extra targets) is we want to have a fully pinned environment for testing purposes.

Reply:
`pip-compile` doesn't play nicely with extras, there's no way to say give me the descendants on [only] the extras. Related comment in a related PR here: https://github.com/apache/incubator-superset/pull/7372#issuecomment-494243758